# This patch file was generated by NetBeans IDE
# Following Index: paths are relative to: /data/www/dartyetvous/cms/src/sites/all/modules/contrib/scald/modules/providers/scald_image
# This patch can be applied using context Tools: Patch action on respective folder.
# It uses platform neutral UTF-8 encoding and \n newlines.
# Above lines and this line are ignored by the patching process.
Index: scald_image.module
--- scald_image.module Base (BASE)
+++ scald_image.module Locally Modified (Based On LOCAL)
@@ -56,7 +56,7 @@
     else {
       $file = file_load($form_state['values']['file']);
     }
-    $atom->title = $file->filename;
+    $atom->title = '';//$file->filename;
     $atom->base_id = $file->fid;
     $langcode = field_language('scald_atom', $atom, 'scald_thumbnail');
     $atom->scald_thumbnail[$langcode][0] = (array) $file;
